challenges/objects-arrays.js

```diff
@@ -5,13 +5,44 @@
   Use this pattern to create your objects: 
   object name, diet, weight, length, period
 */
-
+function Dinosaur(attr) {
+	this.name = attr.name;
+	this.diet = attr.diet;
+	this.weight = attr.weight;
+	this.length = attr.length;
+	this.period = attr.period;
+	this.speak = attr.speak;
+}
+
+Dinosaur.prototype.roar = function() {
+	return `${this.speak}`;
+};
 // tyrannosaurus, carnivorous, 7000kg, 12m, Late Cretaceious
+const tyrannosaurus = new Dinosaur({
+	name: 'Tyrannosaurus',
+	diet: 'carnivorous',
+	weight: '7000kg',
+	length: '12m',
+	period: 'Late Cretaceious',
+});
 
 // stegosaurus, herbivorous, 2000kg, 9m, Late Jurassic
-
+const stegosaurus = new Dinosaur({
+	name: 'Stegosaurus',
+	diet: 'herbivorous',
+	weight: '2000kg',
+	length: '9m',
+	period: 'Late Jurassic',
+});
 // velociraptor, carnivorous, 15kg, 1.8m, Late Cretaceious
-
+const velociraptor = new Dinosaur({
+	name: 'Velociraptor',
+	diet: 'carnivorous',
+	weight: '15kg',
+	length: '1.8m',
+	period: 'Late Cretaceious',
+	speak: 'RAWERSRARARWERSARARARRRR!',
+});
 // Using your dinosaur objects, log answers to these questions:
```

Comment:
So instead of changing all of the objects to Dinosaurs, you could have used prototype to add the roar method to only the tyrannosaurus object:

```
tyrannosaurus.prototype.roar = function() {
	return `${this.speak}`;
};
```